content/en/docs/getting-started/usage/use-openelb-in-vip-mode.md

```diff
@@ -0,0 +1,252 @@
+---
```

Comment:
We might need to add `Use OpenELB in VIP Mode (Beta)` to the catalog.
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/40452856/164675237-5c8c3b0b-ecc0-4098-affe-2eff890057b2.png">

Reply:
It was also missing under `Concepts`, preview it [here](https://website-av0bn79oa-openelb.vercel.app/docs/concepts/).
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/40452856/164675815-58990d38-9e9d-4da3-a52d-a6260866141b.png">